src/clj/game/core/io.clj

```diff
@@ -208,17 +208,17 @@
                                                                             ": " (get-card state target))))
                                            :choices {:req (fn [t] (card-is? t :side %2))}}
                                           {:title "/card-info command"} nil)
-          "/clear-win"  #(clear-win %1 %2)
+          "/clear-win"  clear-win
           "/click"      #(swap! %1 assoc-in [%2 :click] (max 0 value))
-          "/close-prompt" #(command-close-prompt %1 %2)
+          "/close-prompt" command-close-prompt
           "/counter"    #(command-counter %1 %2 args)
           "/credit"     #(swap! %1 assoc-in [%2 :credit] (max 0 value))
           "/deck"       #(toast %1 %2 "/deck number takes the format #n")
           "/discard"    #(toast %1 %2 "/discard number takes the format #n")
           "/discard-random" #(move %1 %2 (rand-nth (get-in @%1 [%2 :hand])) :discard)
           "/draw"       #(draw %1 %2 (max 0 value))
           "/end-run"    #(when (= %2 :corp) (end-run %1 %2))
-          "/error"      #(show-error-toast %1 %2)
+          "/error"      show-error-toast
```

Comment:
I do somewhat miss the symmetry / pattern of the old version... 😛

Reply:
Yeah, either way is fine by me for this one. Makes sense to be explicit, makes sense to curry. I have no preference either way!

Reply:
@nealterrell  is the head honco these days. Any preference?